samples/performance/multithreading_render_passes/multithreading_render_passes.cpp

```diff
@@ -39,6 +39,15 @@ MultithreadingRenderPasses::MultithreadingRenderPasses()
 	config.insert<vkb::IntSetting>(2, multithreading_mode, 2);
 }
 
+void MultithreadingRenderPasses::request_gpu_features(vkb::PhysicalDevice &gpu)
+{
+#ifdef VK_ENABLE_BETA_EXTENSIONS
+	// We need to enable the mutableComparisonSamplers feature of the VK_KHR_portability_subset extension
```

Comment:
... same comment as in async_compute.cpp.

Reply:
Answered in thread above.